Doc/library/tarfile.rst

```diff
@@ -102,6 +102,10 @@ Some facts and figures:
    ``'x:bz2'``, :func:`tarfile.open` accepts the keyword argument
    *compresslevel* (default ``9``) to specify the compression level of the file.
 
+   For modes ``'w:xz'`` and ``'x:xz'``, :func:`tarfile.open` accepts the
+   keyword argument *preset* to specify the compression level of the file. The
+   default is :class:`PRESET_DEFAULT <lzma.LZMACompressor>`.
```

Comment:
`PRESET_DEFAULT` is not a class. You can use the `:data:` role.

Or you can just remove the last sentence, because the default value is the same as in LZMACompressor() and other functions.